Tests/test_SeqIO_Gck.py

```diff
@@ -0,0 +1,339 @@
+# Copyright 2019 Damien Goutte-Gattat.  All rights reserved.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Tests for the SeqIO Gck module."""
+
+from io import BytesIO
+import os.path
+import unittest
+
+from Bio import SeqIO
+
+# The Gck parser has been developed using the files of the
+# Drosophila Gateway Vector Collection from Carnegie Science
+# (<https://emb.carnegiescience.edu/drosophila-gateway-vector-collection>)
+# as sample Gck files. We cannot redistribute those files along with
+# Biopython, therefore we skip the test case by default.
+#
+# To test the Gck parser, download the GCK.zip archive from the URL
+# above and place the files it contains under a 'Tests/Gck' directory.
+@unittest.skipUnless(os.path.exists('Gck'), 'Gck sample files not available')
```

Comment:
That's an elegant solution.

Reply:
Actually maybe we could *automatically* download the archive during the setup phase of the test, if it is not already there? Unless the `--offline` flag has been set of course.

I am testing that right now and it seems to work fine. I can push an update to my branch, but before I do that:

* Do you think it's a good idea?
* If we do download the archive when the tests start, should we remove it when the tests are over? I am inclined to keep it (otherwise it would be downloaded again everytime the tests are run), but then we have to make sure it never finds its way into the repository or a release tarball (easily done by adding an entry to `.gitignore` and `MANIFEST.in`).

Reply:
How big is the download?

If we do this automatically in the test suite, I would also cache the file.

Even if we don't automatically download it, we should add it to the ``.gitignore`` as as a precaution exclude it in ``MANIFEST.in`` as well.

Reply:
The archive is 439 kb.

I agree about the need to prevent the inclusion of any of those files in any case, independently of whether they have been downloaded manually or automatically.